index.js

```diff
@@ -1,58 +1,9 @@
-const registerShortcut = require('hyperterm-register-shortcut');
-const windowSet = new Set([]);
-
-function showWindows (windows, app) {
-  windows.forEach((win, index) => {
-    if (index === windowSet.length-1) {
-      win.focus();
-    } else {
-      win.show();
-    }
-  });
-  if (process.platform !== 'win32') app.show();
-}
+const addWindow = require('./modules/windows').addWindow;
+const setup = require('./modules/setup');
 
-function hideWindows (windows, app) {
-  windows.forEach(win => {
-    if (win.isFullScreen()) return;
-    if (process.platform === 'win32') {
-      win.minimize();
-    } else {
-      win.hide();
-    }
-  });
-  if (process.platform !== 'win32') app.hide(); // Mac OS only (re-focuses the last active app)
-}
-
-function toggleWindowVisibility (app) {
-  const windows = [...windowSet];
-  const focusedWindows = windows.filter(window => window.isFocused());
-  if (focusedWindows.length > 0) {
-    hideWindows(windows, app);
-  } else {
-    showWindows(windows, app);
-  }
-}
-
-function addWindow (window) {
-  windowSet.add(window);
-  window.on('focus', () => {
```

Comment:
Sweet. It's been a while so thanks for calling this out, I might have missed it otherwise